devbox.json

```diff
@@ -4,8 +4,8 @@
     "go@1.22",
     "llvm_14@latest",
     "clang_14@latest",
-    "protobuf@3.24.3",
-    "protoc-gen-go@1.34.1",
+    "protobuf@3.19.4",
```

Comment:
Any reason for the downgrade?

Reply:
For matching protobuf version in main https://github.com/castai/kvisor/blob/main/api/v1/kube/kube_api.pb.go#L4 The version in `devbox.json` mismatches the last version used to generate the protos.

We can have a job in CI that compares generated code with expected generated code to avoid this mismatch in the future but it's not a big problem right now